libsolidity/formal/ModelCheckerSettings.h

```diff
@@ -26,6 +26,30 @@
 namespace solidity::frontend
 {
 
+struct ModelCheckerContracts
+{
+	/// By default all contracts are analyzed.
+	static ModelCheckerContracts All() { return {}; }
+
+	static std::optional<ModelCheckerContracts> fromString(std::string const& _contracts);
+
+	bool all() const { return contracts.empty(); }
```

Comment:
What is this used for? Perhaps comments here would help?

Reply:
It's used by the SMTEncoder to check whether all contracts should be analyzed